nodejs/awsx/cloudwatch/dashboard.ts

```diff
@@ -87,6 +92,9 @@ export class Dashboard extends aws.cloudwatch.Dashboard {
             dashboardName: utils.ifUndefined(args.name, name),
             dashboardBody: getDashboardBody(args).apply(b => JSON.stringify(b)),
         }, opts);
+
+        this.url = pulumi.interpolate
+            `https://${aws.config.region}.console.aws.amazon.com/cloudwatch/home?region=${aws.config.region}#dashboards:name=${this.dashboardName}`;

```

Comment:
Feels like `aws.config.region` is not correct if this dashboard was created with a first class provider that was in a separate region.

Reply:
That's a good point.  Thanks!